src/plugins/console/public/application/models/legacy_core_editor/mode/output_highlight_rules.js

```diff
@@ -25,7 +25,30 @@ export function OutputJsonHighlightRules() {
     },
     {
       token: 'comment',
-      regex: '#.*$',
+      /* Comments start with '#' character and end where characters start with http status and statusText
+       * This will allow us to tokenize the status badges for highlighting the multiple request results in editor output
+       */
+      regex: /#(.*?)(?=\d+\s(?:OK|Bad Request|Not Found|Continue|Created)|$)/,
+    },
+    {
+      token: function (value) {
+        const status = value.match(/\d+/);
+        if (status <= 199) {
+          return 'badge.badge-default';
+        }
+        if (status <= 299) {
+          return 'badge.badge-success';
+        }
+        if (status <= 399) {
+          return 'badge.badge-primary';
+        }
+        if (status <= 499) {
+          return 'badge.badge-warning';
+        }
+        return 'badge.badge-danger';
+      },
+      // Matches any digits ending with http statusText at the end of string
+      regex: /(\d+\s(OK|Bad Request|Not Found|Continue|Created))$/,
```

Comment:
Hi @cjcenizal , I just wanted to check with you. Do we need to localize the HTTP status texts above? (OK, Bad request, etc.)

Reply:
Yes I think we should localize them.

Reply:
Thanks for the suggestions! I've made slight adjustments to the regex pattern to match any digits followed by any letters. WDYT? 

```
(\d+\s[\sA-Za-z]+$)
```

This will avoid matching other numbers at the end of a string in the output panel. E.g. 
<img width="1498" alt="Screen Shot 2022-05-24 at 15 45 29" src="https://user-images.githubusercontent.com/53621505/170018716-5dd506a6-a9ce-4556-a1e5-49729bccdb64.png">